src/App.js

```diff
@@ -1,29 +1,14 @@
-import logo from './logo.svg';
+/* eslint-disable react/prefer-stateless-function */
```

Comment:
- [ ] [OPTIONAL] Please remove the **eslint disable** statement. Because ` Eslint` helps developers to write clean JavaScript codes and adapt best practices when writing the code.  You just need to export it as the default code. :smile: To fix the error add the constructor :+1: 
```
import React from 'react';
import Calculator from './components/Calculator';

class App extends React.Component {
  constructor(props) {
    super(props);
    this.state = {};
  }

  render() {
    return <Calculator />;
  }
}

export default App;

```

Reply:
thank you @codecaiine